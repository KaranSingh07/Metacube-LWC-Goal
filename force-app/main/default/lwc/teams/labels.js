export const LABELS = {
	ToastErrorTitle: 'Error!',
};
