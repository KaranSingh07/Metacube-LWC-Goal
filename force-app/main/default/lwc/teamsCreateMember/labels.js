export const LABELS = {
	ToastSuccessTitle: 'Success!',
	ToastErrorTitle: 'Error!',
	TeamMemberCreated: 'Team member created successfully.',
};
