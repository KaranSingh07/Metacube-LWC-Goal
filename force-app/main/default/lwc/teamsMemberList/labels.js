export const LABELS = {
	ToastSuccessTitle: 'Success!',
	ToastErrorTitle: 'Error!',
	TeamMembersSynced: 'Team members synced successfully.',
	NoTeamSelectedWarning: 'Select a team to show their members.',
	NoTeamMembersExistsWarning: 'This team have no members. Create one from above.',
};
