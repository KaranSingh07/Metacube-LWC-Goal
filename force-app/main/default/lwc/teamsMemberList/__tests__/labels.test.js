import { LABELS } from '../labels';

test('LABELS object should contain relevant values', () => {
	expect(LABELS).toMatchSnapshot();
});
