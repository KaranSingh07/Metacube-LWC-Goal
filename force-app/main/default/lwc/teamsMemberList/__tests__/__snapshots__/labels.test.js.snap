// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LABELS object should contain relevant values 1`] = `
{
  "NoTeamMembersExistsWarning": "This team have no members. Create one from above.",
  "NoTeamSelectedWarning": "Select a team to show their members.",
  "TeamMembersSynced": "Team members synced successfully.",
  "ToastErrorTitle": "Error!",
  "ToastSuccessTitle": "Success!",
}
`;
